bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,74 @@
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours
+from bot.constants import Colours, Emojis
 from bot.decorators import override_in_channel
 
 log = logging.getLogger(__name__)
 
+ICONS = {"ISSUE": Emojis.issue,
+         "ISSUE_CLOSED": Emojis.issue_closed,
+         "PULL_REQUEST": Emojis.pull_request,
+         "PULL_REQUEST_CLOSED": Emojis.pull_request_closed,
+         "MERGE": Emojis.merge}
+
+RESP_VALUE = {404: "Issue/pull request not located! Please enter a valid number!",
+              403: "rate limit has been hit! Please try again later!"}
```

Comment:
```suggestion
              403: "Rate limit has been hit! Please try again later!"}
```
Minor capitalization fix.